### DISCUSSION TITLE
Enhancement: Community Workshop

### DISCUSSION THREAD

**Discussion by SecularSteve** [FunkinCrew/Funkin#1601] [Ideas]
## What is your suggestion, and why should it be implemented?

A workshop containing content made by the community for the community. Ranging anything from Fan Characters, Copy-wrong-ed Content, Total Game Conversions, New Story Modes or even an "Editor's Choice" section containing the best rated of it.

It can be either in the form of a Main Github Web Page, or a direct menu in the game itself titled "workshop" (to signify that it's a separate experience from the official thing).

If the game's going to be pseudo-open source, than you might as well let everybody have easier access to other people's stuff.

As a Newgrounds user since '07, I feel honored by this game. I've just had a feeling somebody had to suggest this sooner or later, but I think everyone needs your opinion on this matter first.

Don't take it like a life-or-death question, it's just a simple suggestion more than anything.

**Comment by TentaRJ** [comment]
GameBanana is a good option...

**Comment by isakube** [comment]
les go Steam API functionality when ninjamuffin

**Reply by steviegt6** [reply]
When the game gets put on Steam...

**Comment by AbnormalPoof** [comment]
Ninjamuffin said that the full-ass game will not have workshop support, so uh
